README.md

```diff
@@ -2,6 +2,20 @@
 
 Terraform module to create VPC Endpoints
 
+## Terraform version requirements
+Terraform `>= 0.12.9` is required due to a bug fix related to empty sets with `for_each`. The fix was included in
+version `0.12.9` -- see the [Changelog here](https://github.com/hashicorp/terraform/blob/v0.12/CHANGELOG.md#0129-september-17-2019).
+The original [bug was reported as issue #22281](https://github.com/hashicorp/terraform/issues/22281).
+
+## Updating documentation
```

Comment:
@lorengordon -- Let me know if this shouldn't be included here. I fumbled with Travis for a bit because I wasn't used to how your team's projects work.

Reply:
Sorry about that, I would have fixed the docs for you. Timing-wise, you got side-swiped by yesterday's terraform-docs update to 0.9.0 where they added the Requirements section. So then the presence of the diff caused the CI failure.

Things get a little touchy to fix yourself, because it's dependent on your local system. Though you clearly figured it out. Either of these should also have worked:

```
## This will install terraform-docs to your local system, may not
## be desirable for you. And may not work if the make target does
## not account for your OS
make docs/generate

## This will run terraform-docs in the docker container, which of
## course requires that you have docker...
make init  # initialize one time...
make docker/run target=docs/generate
```

Reply:
And our Makefiles utilize features of `make` that are only present in fairly recent versions, and not all OS platforms have packages for the latest versions. 🤕 Good times.

Reply:
'tis all good. I was tempted to update the Dockerfile to the latest base image -- but not being familiar with the status of the `tardigrade-ci` project I chose not to.

I'll update the readme to use the make targets!